website/content/partials/post-processor/manifest/Config-not-required.mdx

```diff
@@ -9,7 +9,7 @@
 - `strip_time` (bool) - Don't write the `build_time` field from the output.
 
 - `custom_data` (map[string]string) - Arbitrary data to add to the manifest. This is a [template
-  engine](https://packer.io/docs/templates/legacy_json_templates/engine.html). Therefore, you
+  engine](/packer/docs/templates/legacy_json_templates/engine). Therefore, you
```

Comment:
🤔 Hmm I wonder if we should just not edit this since it seems to be connected with the product output. @ashleemboyer I think that's what's making[ this test fail. ](https://github.com/hashicorp/packer/actions/runs/3972181961/jobs/6809819588)

Reply:
Especially since the comment above says 'DO NOT EDIT MANUALLY -->'  😆

Reply:
Or maybe we update it in the product... I see it here https://github.com/hashicorp/packer/blob/main/post-processor/manifest/post-processor.go#L35

Reply:
I'll ask the packer team

Reply:
That's what I was going to suggest! 👍🏻

Reply:
So I got confirmation to update it in the source. My thought was to use this absolute path, `https://developer.hashicorp.com/packer/docs/templates/legacy_json_templates/engine` but I'm worried the rewrite script will want to make this relative and the tests will fail. Trying to think of the best path forward

Reply:
Perhaps the relative path is okay though, since this comment is meant to be used to generate the docs.

Reply:
I think a relative path will be ok too!